### ISSUE TITLE
Start work on infrastructure

### ISSUE THREAD

**Issue by NotAProton** [tenacityteam/tenacity-legacy#119]
EDIT:  I was out of the loop, work is bieng discussed and done on:
Libera IRC: #tenacity  
Matrix: #tenacity:libera.chat

Issues:
1) Currently there is no place where someone new, wanting to contribute to the project, can get up to speed on the progress.
2) There is no "quick-start guide" for someone who would just like to install Tenacity and be on their way.

Now, I know this project is still in it its very early stages, and you may think its too premature to start work on the forums and guides, but in reality, the support and streamlined installation process is just as important as the software itself.

In the end, the only audacity fork that will survive is the one that can get out of early development and onto the majority of computers, the one which can be installed by the largest amount of people. It will only be then that linux distro maintainers will consider adding Tenacity to their official repos.

My proposal:
 
- Separate teams and leadership for the software and infrastructure teams, the latter managing the website, hosting, IRC chat rooms and other sysadmin stuff. 

- Creation of an easy to navigate website 
Ideal: [Keepassxc](https://keepassxc.org)

- Creation of dedicated forum or well-moderated subreddit, and IRC for general discussion and support.
Ideal: [KDE forum](https://forum.kde.org/) , [KDE Subreddit](https://www.reddit.com/r/kde/) , [Arch IRC](https://wiki.archlinux.org/title/Arch_IRC_channels)

- Simultaneous work on Tenacity and the infrastructure so that the first release can be smooth and successful.
---
Suggested task list:
- [x] Creation of tenacityteam.github.io repo [which will be served on](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site) [tenacityaudio.org](https://tenacityaudio.org)
- [ ] Creation of tenacity-branding repo where contributors can discuss logo ideas and colors.
- [ ] Appointing of infrastructure team.

**Comment by mmahmoudian** [comment]
@emabrey I am a member of [Flameshot](https://github.com/flameshot-org/flameshot) team and am helping with the triage, answering the questions, guiding the community (Github, Slack, Matrix, Weblet), partly handling the documentation ([flameshot.org](https://flameshot.org)), maintaining the AUR and etc.

I will be delighted to also help this project.

**Comment by 12people** [comment]
I'd propose using Matrix instead of IRC. Like IRC, it's decentralized and FOSS, but also provides a wider range of features and Matrix clients tend to be more user friendly.

**Comment by marcthe12** [comment]
I suggest a minimal quick & dirty website. We can make a proper one a little later. Btw are the maintainers own the domain. Just to make sure due to all the drama that occurred recently. To me If we have a domain and a place to download a "safe snapshot" and the ability to do stuff which require reverse dns notation(flatpak packaging). The could allow us to visiable in places other than github.

I suggest matrix as irc is frankly dying and has irc bridges to matrix. I think we can setup forums the last once we get the other major stuff.

**Comment by falkTX** [comment]
IRC is alive and well for linux audio developers at least.
https://netsplit.de/networks/top10.php has some stats on it.

anyway, the chat stuff pretty much already works now.

**Comment by bitdivine** [comment]
Apropos website hosting:
- A github website would probably be the fastest way of getting started, but that is tied to GitHub so is probably not a long term solution.
- Decentralized hosting: The InternetComputer is a decentralized blockchain that hosts websites, amongst other things, and the websites are accessible to normal people with normal browsers.  (Unlike e.g. zeronet which is cool but only accessible to crypto geeks.  Example: [A forum hosted on the Internet Computer](https://h5aet-waaaa-aaaab-qaamq-cai.raw.ic0.app/)).  If such hosting is desired I would be happy to help.

Note: Someone needs to write the website content.  I'll vote myself out of that, if you don't mind!

Note: The Internet Computer governance is based on consensus, upgrades are voted in.  Presumably we would want a set of people who can vote on website updates and, if prebuilt versions of the code are hosted on the website, we would want a voting mechanism or proof that the build matches a suitably tagged version on GitHub.

**Comment by not-a-proton** [comment]
@emabrey @panos Can we open a new repo- tenacityteam.github.io so that people who are ready to start building the website can start opening pull requests.

**Comment by 12people** [comment]
There shouldn't really be a problem with running a static site via Github Pages, as it's trivial to migrate elsewhere if Github stops being a satisfactory host.

(Ironically, using Github itself for e.g. issues is a much bigger problem, as migration to another issue tracker can be painful and problematic.)

**Comment by Be-ing** [comment]
I suggest to use Discourse for a forum. Discourse provides hosting for free for community projects, or it can be self hosted if someone trusted steps up to host it.

**Comment by amrithmmh** [comment]
can we use the same wiki , and docs from audacity and go from there?

**Comment by NotAProton** [comment]
> 
> 
> can we use the same wiki , and docs from audacity and go from there?

Yeah, that makes sense, we will just need install instructions, our docs won't be longer than a few pages.

**Comment by cfanoulis** [comment]
I believe (/hope) fosshost would be willing to support a virtual private server for thks project for hosting forums & stuff

If you need any infra help, I'd also be willing to help ✋

**Comment by RoaddogLabs** [comment]
> I suggest to use Discourse for a forum. Discourse provides hosting for free for community projects, or it can be self hosted if someone trusted steps up to host it.

Discourse is great.  It's my go to.

For the rest of it...

The user base _are not_ code or even tech geeks. They're audio geeks, audiophiles and hobbyists.  Some are pros.  We use it for basic use in high level pro audio productions.  The distribution of the binaries needs to reflect that.  These days that's still a site if you can't get it into an app store.

It's not just this thread but I see a lot of you guys are getting into the weeds with tool and distribution choices.  The quickest fork to ship under the forked name that is as functional as Audacity will have an advantage.  With that the user facing side needs to be ready to roll as soon as the rebrand/fork is ready to release.  You're probably only going to get one shot at non programming/tech users ditching Audacity for the fork.  They want a free (as in beer) editor/recorder, basic DAW.  They don't care about how the sausage is made.  If it's not a solid alternative with a good reason to switch it's game over for any fork that doesn't provide that.

**Comment by JPcode05** [comment]
> > I suggest to use Discourse for a forum. Discourse provides hosting for free for community projects, or it can be self hosted if someone trusted steps up to host it.
> 
> Discourse is great. It's my go to.
> 
> For the rest of it...
> 
> The user base _are not_ code or even tech geeks. They're audio geeks, audiophiles and hobbyists. Some are pros. We use it for basic use in high level pro audio productions. The distribution of the binaries needs to reflect that. These days that's still a site if you can't get it into an app store.
> 
> It's not just this thread but I see a lot of you guys are getting into the weeds with tool and distribution choices. The quickest fork to ship under the forked name that is as functional as Audacity will have an advantage. With that the user facing side needs to be ready to roll as soon as the rebrand/fork is ready to release. You're probably only going to get one shot at non programming/tech users ditching Audacity for the fork. They want a free (as in beer) editor/recorder, basic DAW. They don't care about how the sausage is made. If it's not a solid alternative with a good reason to switch it's game over for any fork that doesn't provide that.

Audacity Forum uses phpBB instead of Discourse. To make it easier for those users (and for me), and since Discourse costs money (Edit: I just realised they provide free hosting for projects like this, didn't realize), why not use phpBB?

**Comment by Be-ing** [comment]
phpBB is ancient code and UI design. Have you ever tried using a phpBB forum on a smartphone? It's bad. It's also a magnet for spam, speaking from experience of moderating a phpBB forum.

**Comment by RoaddogLabs** [comment]
I've been hosting and working on projects that use Discourse for a few years now.  The downside is that it's much more complex to install and configure than phpBB but it offers far more in terms of performance to resource and has features others don't.  Under the hood it's built to be more secure than a PHP forum and is build to massively scale 

The goal needs to be getting something user facing at the time of launch.  If it's phpBB or any of the others that's fine.  The point is there needs to be something.

The Audacity docs are CC with attribution so those are fair game as long as the CC terms are followed.

**Comment by Be-ing** [comment]
Discourse's anti-spam and moderation features are great IME.

**Comment by JPcode05** [comment]
> phpBB is ancient code and UI design. Have you ever tried using a phpBB forum on a smartphone? It's bad. It's also a magnet for spam, speaking from experience of moderating a phpBB forum.

> I've been hosting and working on projects that use Discourse for a few years now. The downside is that it's much more complex to install and configure than phpBB but it offers far more in terms of performance to resource and has features others don't. Under the hood it's built to be more secure than a PHP forum and is build to massively scale

> Discourse's anti-spam and moderation features are great IME.

Then we will use Discourse.

**Comment by NotAProton** [comment]
Libera IRC: #tenacity
Matrix: #tenacity:libera.chat

**Comment by HumbleDeer** [comment]
> phpBB is ancient code and UI design. Have you ever tried using a phpBB forum on a smartphone? It's bad. It's also a magnet for spam, speaking from experience of moderating a phpBB forum.

Themes are still very much a good way of theming PhpBB. It's the Prosilver theme that's out of date. That doesn't mean by any stretch that all of PhpBB is out of date.

**Comment by RoaddogLabs** [comment]
> Themes are still very much a good way of theming PhpBB. It's the Prosilver theme that's out of date. That doesn't mean by any stretch that all of PhpBB is out of date.

There are better technologies than PHP for hosting that sort of app these days.  They’re lighter, faster and have the ability to scale better right out of the box.  While phpBB can do mobile Discourse provides a lighter, faster data fetch and render.  It’s using Redis which speeds retrieval of commonly used data sets cached in memory as key pairs without having to hit the database. The price for these benefits is the tradeoff in complexity for those supporting the backend.  Users and mods don’t need to be concerned with that particularly if Discourse hosts it.

**Comment by nm17** [comment]
As already said, Discorse is the only real option for a community forum nowadays.
